libpromises/unix.c

```diff
@@ -247,7 +247,16 @@ static bool GetUserGroupInfoFromGetent(const char *type, const char *query,
                                        LogLevel error_log_level)
 {
     char buf[CF_BUFSIZE];
-    NDEBUG_UNUSED int print_ret = snprintf(buf, sizeof(buf), "/bin/getent %s %s", type, query);
+    struct stat sb;
+    char* getent_bin;
+
+    if (stat("/bin/getent", &sb) == 0)
+    {
+        getent_bin = "/bin/getent";
+    } else {
```

Comment:
@vpodzime, maybe you could make the curly braces conform with our guideline https://github.com/cfengine/core/blob/master/CONTRIBUTING.md#formatting--whitespace while you're at it?
```suggestion
    }
    else
    {
```